portfolio/src/main/webapp/script.js

```diff
@@ -32,4 +32,93 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+google.charts.load('current', {'packages': ['sankey']});
+google.charts.setOnLoadCallback(drawChart);
 
+// google.charts.load('current', {'packages':['sankey']});
```

Comment:
Please delete this commented code! It'd be better to put it in another development branch if the plan is to keep working on it

Reply:
My apologies! I forgot to remove it, but I'll do so now.